src/Xamarin.Android.Build.Tasks/Tasks/JavaToolTask.cs

```diff
@@ -216,8 +218,12 @@ protected void AppendTextToErrorText (string text)
 		protected override void LogEventsFromTextOutput (string singleLine, MessageImportance messageImportance)
 		{
 			errorLines.Add (singleLine);
-			base.LogEventsFromTextOutput (singleLine, messageImportance);  // not sure why/when we would skip this?
-
+			
+			if (CallBaseLogEventsFromTextOutput)
```

Comment:
I think it violates LSP. Can we just add new protected virtual method, say, LogEvents(...) and override it in D8/R8 subclasses?